### DISCUSSION TITLE
Add the Keyboard Layout New AZERTY (AFNOR)

### DISCUSSION THREAD

**Discussion by Neocamelus** [linuxdeepin/developer-center#3470] [Features Request & Ideas | 特性请求 & 头脑风暴]
I have a French AZERTY keyboard 105 keys With Euro.
It is a keyboard that respects the new Afnor AZERTY standard ( [NF Z71-300](https://norme-azerty.fr/en/) )

I would like to use my keyboard with Deepin 20.7 but only the traditional Azerty is offered.

Here is its configuration in XFree86 under Debian:
X11 Layout: fr
X11 Model: pc105
X11 Variant: afnor

Please, can you add the FRENCH AFNOR Standardized AZERTY layout which is present in the XKB-DATA package in Debian?

**Comment by tsic404** [comment]
deepin 20 based on debian, this keyboard?
![image](https://user-images.githubusercontent.com/48748112/196836116-01d275e4-3a7b-4e8a-8790-59d31cb6ee73.png)

**Comment by Neocamelus** [comment]
Yes, that's the good keyboard.
What program are you using?

**Reply by BLumia** [reply]
If you refer to the screenshot that @tsic404 provided, it's the default "Control Center" app as a part of DDE.

**Comment by Neocamelus** [comment]
For me it is like this: 
![Control Center](https://user-images.githubusercontent.com/12272459/198981053-8faf177e-ae90-498a-b01f-e29d56ab1497.png)


It must be another version of Deepin or there is an additional package.
Is this a beta version? I have version 20.7.1

**Reply by tsic404** [reply]
It seems that fcitx lost this keyboard layout
